tests/foreman/ui/test_errata.py

```diff
@@ -757,3 +762,97 @@ def test_positive_filtered_errata_status_installable_param(session, errata_statu
                 if key in expected_values
             }
             assert actual_values == expected_values
+
+
+@tier3
+def test_content_host_errata_search_commands(session, module_org, module_repos_col):
+    """ View a list of affected content hosts for security (RHSA) and bugfix (RHBA) errata,
+    filtered with errata status and applicable flags. Applicability is calculated using the
+    Library, but Installability is calculated using the attached CV, and is subject to the
+    CV's own filtering.
+
+:id: 45114f8e-0fc8-4c7c-85e0-f9b613530dac
+
+:Setup: Two Content Hosts, one with RHSA and one with RHBA errata.
+
+:Steps:
+    1.  host list --search "errata_status = security_needed"
+    2.  host list --search "errata_status = errata_needed"
+    3.  host list --search "applicable_errata = RHSA-2012:0055"
+    4.  host list --search "applicable_errata = RHBA-2012:1030"
+    5.  host list --search "applicable_rpms = walrus-5.21-1.noarch"
+    6.  host list --search "applicable_rpms = kangaroo-0.3-1.noarch"
+    7.  Create filter & rule to hide RPM (Applicable vs. Installable test)
+    8.  Repeat steps 3 and 5, but 5 expects host name not found.
```

Comment:
Is this relevant to bugzilla at hand?
Don't we cover similar scenarios elsewhere?

Reply:
I can remove the bugzilla number if you prefer?
 I cannot find any test that checks Applicable vs. Installable in the sense of an erratum is not installable, because its not in the CV, but it is still shown as Applicable (its in the Library and the host needs it).

Reply:
@mirekdlugosz , I did add this test in CLI section recently: https://github.com/SatelliteQE/robottelo/blob/9b80440f0df4662f5383f278a8d759b666b7fff9/tests/foreman/cli/test_errata.py#L578

So I can remove those last two steps from this test.

Reply:
@swadeley I think you have removed this filtering part from test itself, so it's only matter of updating steps in docstring now.

Reply:
Right, thanks.